ports/glutin/window.rs

```diff
@@ -1268,12 +1269,28 @@ impl WindowMethods for Window {
                     self.event_queue.borrow_mut().push(WindowEvent::Reload);
                 }
             }
+            //On windows 8.1 when pressing ctrl + anything, ch = Some('\u{unicode_value}')
+            //so matching ch with Some('l') doesn't work here
+            //Instead we match with what's inside the key variable
+            (CMD_OR_CONTROL, _, Key::L) => {
+                if let Some(true) = PREFS.get("shell.builtin-key-shortcuts.enabled").as_boolean() {
+                    //TODO: 
+                    // 1) make it work when clicking on "validate" button - Currently only works when pressing enter
+                    // 2) Erratic behavior - sometime the dialog reloads empty after pressing enter
+                    // (windows 8.1)
+                    if let Some(user_input) = tinyfiledialogs::input_box("Enter a URL", "URL:", "")  {
```

Comment:
The default value should be the current url, not `""`.

I'm wondering if we have any locales support. I'm not sure we want to keep english sentences in the code.